pages/api/v1/reset-account.ts

```diff
@@ -0,0 +1,83 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { prismaClient } from "../../../lib/prisma-client";
+import { getToken } from "next-auth/jwt";
+import { main } from "../../../lib/seeds/main";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  console.log("/reset-account", req.body);
+
+  const token = await getToken({
+    req: req,
+  });
+  // console.log("gSSP token", token);
+
+  if (!token?.sub) {
+    throw new Error("No session");
+  }
+
+  const deleteSpaces = await prismaClient.space.deleteMany({
+    where: {
+      userId: token.sub,
+    },
+  });
+
+  const deleteActions = await prismaClient.action.deleteMany({
+    where: {
+      userId: token.sub,
+    },
+  });
+
+  const user = await prismaClient.user.findUnique({
+    where: { id: token.sub },
+    include: { organization: true },
+  });
+  const organizationId = user?.organizationId;
+  const employees = await prismaClient.employee.findMany({
+    where: { organizationId },
+  });
+
+  const deleteEmployeeBenefitPlan =
+    await prismaClient.employeeBenefitPlan.deleteMany({
+      where: {
+        employeeId: {
+          in: employees.map((e) => e.id),
+        },
+      },
+    });
+
+  const deleteEmployees = await prismaClient.employee.deleteMany({
+    where: {
+      organizationId,
+    },
+  });
+
+  const deleteJobs = await prismaClient.job.deleteMany({
+    where: {
+      organizationId,
+    },
+  });
+
+  const deleteBenefits = await prismaClient.benefitPlan.deleteMany({
+    where: {
+      organizationId,
+    },
+  });
+
+  const runResetSeedFile = async () => {
+    await main();
+  };
+
+  runResetSeedFile();
```

Comment:
I was trying to be descriptive. Added a comment instead. 

The pause is because if I move very fast, I can get to the employees or jobs page and the seed hasnt completed yet. There would be no data there.